Content.Server/Worldgen/Systems/Debris/DebrisFeaturePlacerSystem.cs

```diff
@@ -154,6 +154,8 @@ private void OnDebrisShutdown(EntityUid uid, OwnedDebrisComponent component, Com
                 .Where(x => !Deleted(x.Value))
                 .Select(static x => x.Key)
                 .ToList();
+
+            component.OwnedDebris.Clear();
```

Comment:
I'm not actually sure if this is the right fix, I'm not sure what this system is doing or why its re-spawning existing entities?